docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1269 @@
+        the cloud provider that can be exposed through these persistent volume
+        claims.</para>
+
+        <para>Persistent Volume Claims can be created by overriding the
+        storage values in the delivered Helm chart. The values in the
+        <emphasis
+        role="boldIt">examples/local/values-localfile.yaml</emphasis> provided
+        override the corresponding entries in the original delivered stock
+        HPCC Systems helm chart. The localfile chart creates persistent
+        storage volumes. You can use the
+        <emphasis>values-localfile.yaml</emphasis> directly (as demonstrated
+        in separate docs/tutorials) or you can use it as a basis for creating
+        your own override chart.</para>
+
+        <para>To define a storage plane that utilizes a PVC, you must decide
+        on where that data will reside. You create the storage directories,
+        with the appropriate names and then you can install the localfiles
+        Helm chart to create the volumes to use the local storage option, such
+        as in the following example:</para>
+
+        <programlisting>helm install mycluster hpcc/hpcc -f examples/local/values-localfile.yaml</programlisting>
+
+        <para>Note: The settings for the PVC's must be ReadWriteMany, except
+        for Dali which can be ReadWriteOnce.</para>
+
+        <para>There are a number of resources, blogs, tutorials, even
+        developer videos that provide step-by-step detail for creating
+        persistent storage volumes.</para>
+      </sect3>
+
+      <sect3 id="CYML_BareMEtalStorage">
+        <title>Bare Metal Storage</title>
+
+        <para>There are two aspects to using bare metal storage in the
+        Kubernetes system. The first is the <emphasis>hostGroups</emphasis>
+        entry in the storage section which provides named lists of hosts. The
+        <emphasis>hostGroups</emphasis> entries can take one of two forms.
+        This is the most common form, and directly associates a list of host
+        names with a name:</para>
+
+        <programlisting>storage: 
+  hostGroups: 
+  - name:  &lt;name&gt; "The name of the host group" 
+    hosts: [ "a list of host names" ] 
+</programlisting>
+
+        <para>The second form allows one host group to be derived from
+        another:</para>
+
+        <programlisting>storage: 
+  hostGroups: 
+  - name: "The name of the host group process" 
+    hostGroup: "Name of the hostgroup to create a subset of" 
+    count: &lt;Number of hosts in the subset&gt; 
+    offset: &lt;the first host to include in the subset&gt;  
+    delta: &lt;Cycle offset to apply to the hosts&gt; 
+</programlisting>
+
+        <para>Some typical examples with bare-metal clusters are smaller
+        subsets of the host, or the same hosts, but storing different parts on
+        different nodes, for example:</para>
+
+        <programlisting>storage: 
+  hostGroups: 
```

Comment:
future: it feels like this (and probably other specialist areas), should be rerefenced, but not in this main doc. @ghalliday what do you think

Reply:
Yes.  Need two different documents really.  An introduction - which this is, and a reference guide, which would go through the options in detail.